api/config/authz/config_request.go

```diff
@@ -47,8 +47,13 @@ func (c *ConfigRequest) Validate() error {
 	projectLimit := c.GetV1().GetSys().GetService().GetProjectLimit()
 
 	if projectLimit != nil {
-		if limit := projectLimit.GetValue(); limit < constants_v2.MaxProjects {
-			e := errors.Errorf("project limit must be at least %v", constants_v2.MaxProjects)
+		if limit := projectLimit.GetValue(); limit < constants_v2.MinConfigurableProjects {
+			// Previously, users could not configure their project_limit to be below the
+			// default.
+			// MinConfigurableProjects supports customers who previously
+			// increased their limit to a number that is lower than the new default.
+			// Can be removed when we no longer limit projects.
+			e := errors.Errorf("project limit must be at least %v", constants_v2.MinConfigurableProjects)
 			cfgErr.AddInvalidValue("auth_z.v1.sys.service.project_limit", e.Error())
```

Comment:
[supernit] Looks like you are just passing a string (`e.Error()`) so you don't need an error at all:

```
failureStr := fmt.Sprintf("project limit must be at least %d", constants_v2.MinConfigurableProjects)
cfgErr.AddInvalidValue("auth_z.v1.sys.service.project_limit", failureStr)
```

Reply:
Huh. Good catch.